game_api.txt

```diff
@@ -155,7 +155,7 @@ The doors mod allows modders to register custom doors and trapdoors.
 ### Fence gate definition
 
 	description = "Wooden Fence Gate",
-	texture = "default_wood.png",
+	texture = "default_wood.png", -- `backface_culling` will automatically be set to `true` if it's not given.
```

Comment:
Since line 230 has 113 columns (with tabsize 4) I thought that code style wouldn't apply in `game_api.txt` and this line with 110 columns would be ok.